website/docs/sdk-reference/node.md

```diff
@@ -82,54 +87,91 @@ _ConfigCat Client_ is responsible for:
 
-`createClientWithAutoPoll()`, `createClientWithLazyLoad()`, `createClientWithManualPoll()`  
-Creating the client is different for each polling mode.
-[See below](#polling-modes) for details.
+Options also include a property named `setupHook`, which you can use to subscribe to the hooks (events) of the SDK at initialization time. [More about hooks](#hooks).
+
+For example:
+
+```js
+const configCatClient = configcat.getClient('#YOUR-SDK-KEY#', configcat.PollingMode.AutoPoll, {
+  setupHooks: hooks => hooks.on('clientReady', () => console.log('Client is ready!'))
+});
+```
 
 :::caution
 We strongly recommend you to use the _ConfigCat Client_ as a Singleton object in your application.
-If you want to use multiple SDK Keys in the same application, create only one _ConfigCat Client_ per SDK Key.
+You can acquire singleton client instances for your SDK keys using the `configcat.getClient('<sdkKey>')` function.
+(However, please keep in mind that subsequent calls to `getClient()` with the *same SDK Key* return a *shared* client instance, which was set up by the first call.)
+You can close all open clients at once using the `configcat.disposeAllClients()` function or do it individually using the `configCatClient.dispose()` method.
 :::
 
-## Anatomy of `getValue()`
+## Anatomy of `getValueAsync()`
+
+Returns a Promise with the value.
 
 | Parameters     | Description                                                                                                  |
 | -------------- | ------------------------------------------------------------------------------------------------------------ |
-| `key`          | **REQUIRED.** Setting-specific key. Set on _ConfigCat Dashboard_ for each setting.                           |
+| `key`          | **REQUIRED.** The key of a specific setting or feature flag. Set on _ConfigCat Dashboard_ for each setting.  |
 | `defaultValue` | **REQUIRED.** This value will be returned in case of an error.                                               |
-| `callback`     | **REQUIRED.** Called with the actual setting value.                                                          |
 | `user`         | Optional, _User Object_. Essential when using Targeting. [Read more about Targeting.](advanced/targeting.md) |
 
 ```js
-configCatClient.getValue(
+const value = await configCatClient.getValueAsync(
   'keyOfMySetting', // Setting Key
   false, // Default value
-  (value) => {
-    console.log(value);
-  }, // Callback function
   { identifier: '435170f4-8a8b-4b67-a723-505ac7cdea92' }, // Optional User Object
 );
 ```
 
-## Anatomy of `getValueAsync()`
+or
 
-Returns a Promise with the value.
+```js
+configCatClient
+  .getValueAsync(
+    'keyOfMySetting', // Setting Key
+    false, // Default value
+    { identifier: '435170f4-8a8b-4b67-a723-505ac7cdea92' },
+  ) // Optional User Object
```

Comment:
this comment is in the wrong line, does the formatter allow it in the above line?

Reply:
Sharp eyes! 👍 Fixed by https://github.com/configcat/docs/pull/195/commits/697074f1f6153e1a6b3ef36caee4338cace795cb

On top of that, I also revised `User` object creation as the original code examples weren't completely correct. (Although `const user: User = { ... }` works, the created object won't have the correct prototype set which could lead to hard to track issues. Not in this case but I think it's better to advertise the correct usage.)

Reply:
Absolutely yes! Thanks for bringing it up.